app/isomorphic/components/molecules/SocialLogin/index.js

```diff
@@ -1,102 +1,94 @@
-// import React, { useState } from "react";
-import React from "react";
-import PT from "prop-types";
-// import get from "lodash/get";
-// import { WithFacebookLogin, WithGoogleLogin /* WithTwitterLogin */ } from "@quintype/components";
-// import { Button } from "../../atoms/button";
+import React, { useState } from "react";
+import { func } from "prop-types";
+import { withFacebookLogin, withGoogleLogin } from "@quintype/bridgekeeper-js";
+
 import { FbIcon } from "../../atoms/icons/fb-icon";
-// import { TwitterIcon } from "../../atoms/icons/twitter-icon";
 import { Google } from "../../atoms/icons/google";
-// import IntegrationData from "../../../../../config/integrations.js";
+import Button from "../../atoms/Button";
+
 import "./social-login.m.css";
 
 export const SocialLogin = ({ checkForMemberUpdated }) => {
-  // const [error, setError] = useState("");
-  // const { appKeys } = IntegrationData();
-  // const facebookKey = get(appKeys, ["facebookAppkey"], "");
-  // const googleClientId = get(appKeys, ["googleClientId"], "");
-  // const twitterKey = get(appKeys, ["twitterAppkey"], "");
+  const [error, setError] = useState("");
+
+  const socialLogin = (e, login) => {
+    e.preventDefault();
+
+    login()
+      .then(() => {
+        checkForMemberUpdated();
+        checkForMemberUpdated().then(res => {
```

Comment:
why we need to call 2 times here, we can just it by line num 20 i think .

Reply:
sure